modules/local/show_supported_models.nf

```diff
@@ -21,7 +21,7 @@ process SHOW_SUPPORTED_MODELS {
     "${task.process}":
         mhcflurry: \$(echo \$(mhcflurry-predict --version 2>&1 | sed 's/^mhcflurry //; s/ .*\$//') )
```

Comment:
Same here

Reply:
I thought it would still be good to print the versions here.